.github/workflows/docker.yml

```diff
@@ -11,14 +11,20 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        distro: ["scratch", "alpine", "debian"]
+        distro:
+          - scratch
+          - alpine
+          - debian
```

Comment:
What do you think about keeping entries sorted here alphabetically?

The order does not matter, so alphabetically makes the most sense.